src/Components/Bookings.js

```diff
@@ -1,9 +1,22 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
 import SearchResults from "./SearchResults.js";
-import FakeBookings from "../data/fakeBookings.json";
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
+
+  function fetchData(url) {
+    fetch(url)
+      .then(response => response.json())
+      .then(data => {
+        setBookings(data);
+      })
+      .catch(err => console.log(err));
+  }
+  useEffect(() => {
+    fetchData("https://cyf-react.glitch.me");
+  }, []);
```

Comment:
This looks good, but I'm not sure why we need the `fetchData` function to be separate? This might be useful in the future if we were fetching data for multiple components, but at the moment we only have 1, so generally I'd expect it to be inline in the `useEffect` callback function. Just a minor point though this looks good 👍

Reply:
Thank you. I modified.